### ISSUE TITLE
Browser support level doc

### ISSUE THREAD

**Issue by joolswood** [AusDTO/gov-au-ui-kit#431]
Does the BROWSER-SUPPORT.md and individual component content meet users needs around level of browser support?
As a UI-KIT team member I need to know if we are meeting our users' needs around the browser support level of all parts of the framework - so we know how to improve our documentation.

Reference: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices <-- maybe we should consider `compliant` and `functional` as other categories?

**Comment by klepas** [comment]
Picking up where you left off from @joolswood (https://github.com/AusDTO/gov-au-ui-kit/pull/465#issuecomment-276259781): agreed.

IMHO we should present this information directly in the design guide, as page unto itself.

I’d suggest that any research would probably support that, but happy to test it/put an inquiry out there amongst our user base.

**Comment by joolswood** [comment]
Yes I think it would, but I'm also interested to know other ways that people would find this information valuable; ie do we go back to having support level listed on each individual component.

Maybe getting too down in the weeds with this, there's plenty of other stuff we need user research support to investigate.

**Comment by klepas** [comment]
> do we go back to having support level listed on each individual component.

This is very viable in 2.x, given we’re going modular — makes it technically easier, and also easier to discretely associate granular browser stuff for each component…

… that said, mixing components together can still produce undesired, difficult-to-predict results.

**Comment by joolswood** [comment]
I'm happy to:
- leave the UR work until v2
- reproduce the table and minimum explanatory text, for now.

**Comment by klepas** [comment]
I’d currently opt for the former. I think @dominikwilkowski would agree.

We leave this issue open though, right? :)

**Comment by joolswood** [comment]
Oh I was recommending we did both @klepas - I'll get on to the table.
